swift/README.md

```diff
@@ -1,35 +1,39 @@
+* **Cross-service examples** - Sample applications that work across multiple AWS services.
 
-## Running the code
+### Finding code examples
+Single-service actions and scenarios are organized by AWS service in the
+[example_code](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/swift/example_code/)
+directory. A README in each folder lists and describes how to run the examples.
 
-See the individual readme files in each service directory for information about specific code examples for that service.
+There are currently no cross-service examples for the AWS SDK for Swift. To request a cross-service example, create an issue in the [AWS SDK Code Examples](https://github.com/awsdocs/aws-doc-sdk-examples/) repo.
 
-## Notes
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+*  We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
-- We recommend that you grant this code least privilege,
-  or at most the minimum permissions required to perform the task.
-  For more information, see
-  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
-  in the AWS Identity and Access Management User Guide.
-- This code has not been tested in all AWS Regions.
-  Some AWS services are available only in specific
-  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
-- Running this code might result in charges to your AWS account.
+### Prerequisites
+* You must have an AWS account, and have configured your default credentials and AWS Region as described in the [Getting started section](https://docs.aws.amazon.com/sdk-for-swift/latest/developer-guide/getting-started.html) of the _AWS SDK for Swift Developer Guide_.
+* You must have the [Swift](https://www.swift.org/) compiler (version 5.4 or later) and tools installed. If you have the latest version of Xcode installed, you already have the Swift tools. See [Setting up in the AWS SDK for Swift Developer Guide](https://docs.aws.amazon.com/sdk-for-swift/latest/developer-guide/setting-up.html) for  detailed instructions.
 
-## For more information
+## Tests
+**Note**: Running the tests might result in charges to your AWS account.
 
+To run the tests for an example, use the command `swift test` in the example's directory.
+
+## Additional resources
 * [AWS SDK for Swift Developer Guide](https://docs.aws.amazon.com/sdk-for-swift/latest/developer-guide): The documentation for the AWS SDK for Swift
```

Comment:
* [AWS SDK for Swift Developer Guide](https://docs.aws.amazon.com/sdk-for-swift/latest/developer-guide) - Documentation for the AWS SDK for Swift